HW02/Kate_Fedotova/kate_fedotova_hw_02_t_03.rb

```diff
@@ -0,0 +1,22 @@
+require 'time'
+
+puts 'File open: task_2.txt' if File.open('task_2.txt')
+
+abort 'File is not exist!' unless File.exist?('task_2.txt')
+
+logs = File.readlines('task_3.txt')
+
+def get_events_interval(lines)
+  array = []
+  valid_events = lines.select { |line| line.include?('Calling core with') }
+  return [0] if valid_events.size <= 1
+
+  valid_events.each_cons(2) do |first, second|
```

Comment:
Could you try using `.map` here?
```suggestion
  valid_events.each_cons(2).map do |first, second|
```

Reply:
done)